### ISSUE TITLE
Warning shows after UI input request on apply

### ISSUE THREAD

**Issue by pselle** [hashicorp/terraform#26736]
### Terraform Version

```
v0.14.0-dev
```

### Terraform Configuration Files

```terraform
variable "cat" {
  default   = "cat"
  sensitive = true
}

resource "random_pet" "bar" {
  prefix = var.cat
}

terraform {
  experiments = [provider_sensitive_attrs]
}

```

### Debug Output

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # random_pet.bar will be created
  + resource "random_pet" "bar" {
      + id        = (known after apply)
      + length    = 2
      + prefix    = (sensitive)
      + separator = "-"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: 
Warning: Experimental feature "provider_sensitive_attrs" is active

  on main.tf line 52, in terraform:
  52:   experiments = [provider_sensitive_attrs]
```

### Expected Behavior

The warning should not displace the UI input, that is, your "yes" should be typed on the same line as the prompt.

### Steps to Reproduce

1. terraform init
2. terraform apply

**Comment by jbardin** [comment]
The primary culprit here is `panicwrap`. While the process output streams are not technically synchronized in any way, stderr is typically unbuffered and should not be delayed when writing the warning output. This means writing to stderr just before stdout will typically also write to the terminal in that order.

The `panicwrap` package however intercepts stderr, buffers it, and parses it searching for text that looks like a panic. This added delay is what is causing the warnings to often show up after the stdout text, which can be confirmed by running terraform with `TF_FORK=0`.  

The only choices I see here are either writing all warnings to stdout rather than stderr, or dropping  `panicwrap` altogether.

**Comment by pselle** [comment]
This has been fixed at some point in the 0.14 beta cycle, after a re-test:

<img width="750" alt="Screen Shot 2020-11-30 at 3 00 27 PM" src="https://user-images.githubusercontent.com/204372/100658515-2e5f5580-331d-11eb-8838-21e8592b8c30.png">

The warning shows, but doesn't disrupt the flow of the UI input request.
